components/server/src/api/server.ts

```diff
@@ -159,8 +159,10 @@ export class API {
             get(target, prop) {
                 return (...args: any[]) => {
                     const connectContext = args[1] as HandlerContext;
+                    const requestId = connectContext.requestHeader.get("x-request-id") || v4();
```

Comment:
Hm. It feels like we should separate these two IDs: The one we use internally should be :100: under our control. And if we receive one from the client that's cool, we can add it as separate field.